ios and windows builds are failing for some reason, restart does not help @rasom 

```
09:37:24  warning "react-navigation > create-react-context@0.2.2" has unmet peer dependency "prop-types@^15.0.0".
09:39:02  [4/4] Building fresh packages...
09:39:02  success Saved lockfile.
09:39:02  $ yarn build
09:39:02  yarn run v1.13.0
09:39:02  $ babel --no-babelrc --plugins=syntax-jsx,syntax-class-properties,syntax-object-rest-spread,transform-flow-strip-types src --copy-files --out-dir dist --ignore '**/__tests__/**'
09:39:02  internal/modules/cjs/loader.js:582
09:39:02      throw err;
09:39:02      ^
09:39:02  
09:39:02  Error: Cannot find module 'core-js/library/fn/object/keys'
09:39:02      at Function.Module._resolveFilename (internal/modules/cjs/loader.js:580:15)
09:39:02      at Function.Module._load (internal/modules/cjs/loader.js:506:25)
09:39:02      at Module.require (internal/modules/cjs/loader.js:636:17)
09:39:02      at require (internal/modules/cjs/helpers.js:20:18)
09:39:02      at Object.<anonymous> (/Users/jenkins/Library/Caches/Yarn/v4/.tmp/2c1913c5afdf8957a3b46d6cd09fe0ce.2f1f8cd782ffcba362ae333e3dc630e237e97669.prepare/node_modules/babel-runtime/core-js/object/keys.js:1:93)
09:39:02      at Module._compile (internal/modules/cjs/loader.js:688:30)
09:39:02      at Object.Module._extensions..js (internal/modules/cjs/loader.js:699:10)
09:39:02      at Module.load (internal/modules/cjs/loader.js:598:32)
09:39:02      at tryModuleLoad (internal/modules/cjs/loader.js:537:12)
09:39:02      at Function.Module._load (internal/modules/cjs/loader.js:529:3)
09:39:02  error Command failed with exit code 1.
09:39:02  info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
09:39:02  error Command failed with exit code 1.
09:39:02  info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
09:39:02  make: *** [prepare-ios] Error 1
```

@jakubgs

--- a/mobile_files/package.json.orig
+++ b/mobile_files/package.json.orig
@@ -61,6 +61,7 @@
     "react-native-udp": "git+https://github.com/status-im/react-native-udp.git#2.3.1-1",
     "react-native-webview-bridge": "git+https://github.com/status-im/react-native-webview-bridge.git#0.33.16-4",
     "react-navigation": "^2.12.1",
+    "react-navigation-stack": "git+https://github.com/status-im/react-navigation-stack.git#0.7.0",
     "realm": "2.21.0",
     "rn-snoopy": "git+https://github.com/status-im/rn-snoopy.git#v2.0.2-status",
     "string_decoder": "0.10.31",

